src/foam/u2/wizard/wizardlet/ReviewWizardletView.js

```diff
@@ -84,6 +87,9 @@ foam.CLASS({
                   })
                   .start(item.headingBorder)
                     .tag(item.view, {data: self.data.value[item.name]})
+                    .startContext({ data: self.data?.value?.[item?.name] })
+                      .tag(item.view)
+                    .endContext()
```

Comment:
This seems to be breaking a few wizardlets, below suggestion should fix that
```suggestion
                    .tag(item.view, { ...(
                      self.data.value[item.name] ?
                      {data: self.data.value[item.name]} :
                      {}
                    )})
```